modules/nw-sriov-add-pod-runtimeconfig.adoc

```diff
@@ -99,3 +99,103 @@ $ oc create -f <filename> <1>
 ----
 $ oc exec sample-pod -n <namespace> -- ip addr show
 ----
+
+= Configuring static GUID and IP addresses on additional SR-IOV InfiniBand networks
+
+You can configure static Global Unique Identifiers (GUID) and IP addresses on an SR-IOV InfiniBand network by specifying Container Network Interface (CNI) runtimeConfig data in a Pod annotation.
+
+.Prerequisites
+
+* Install the {product-title} Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges when creating the SriovNetwork CR.
+
+.Procedure
+
+. Create the following SriovIBNetwork CR, and then save the YAML in the `<name>-sriov-ib-network.yaml` file. Replace `<name>` with a name for this additional network.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovIBNetwork
+metadata:
+  name: <name> <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  networkNamespace: <target_namespace> <3>
+  ipam: '{ "type": "static" }' <4>
+  capabilities: '{ "infiniband_guid": true, "ips": true }' <5>
+  resourceName: <sriov_resource_name> <6>
+----
+<1> Replace `<name>` with a name for the CR. The SR-IOV Network Operator creates a NetworkAttachmentDefinition CR with same name.
+<2> Specify the namespace where the SR-IOV Network Operator is installed.
+<3> Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR is created.
+<4> Specify static type for the ipam CNI plug-in as a YAML block scalar.
+<5> Specify `infiniband_guid` and `ips` `capabilities` to `true`.
+<6> Replace `<sriov_resource_name>` with the value for the `spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
+
+. Create the CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <filename> <1>
+----
+<1>  Replace `<filename>` with the name of the file you created in the previous step.
+
+. Optional: Confirm that the NetworkAttachmentDefinition CR associated with the SriovIBNetwork CR that you created in the previous step exists by running the following command. Replace `<namespace>` with the namespace you specified in the SriovIBNetwork CR.
```

Comment:
Replace `<namespace>` with the namespace you specified in the SriovIBNetwork CR
better use `with the networkNamespace you specified` in order to avoid confusion with openshift-sriov-network-operator` namespace`

Reply:
Done